src/app/game-state/main-loop.service.ts

```diff
@@ -98,6 +99,10 @@ export class MainLoopService {
       this.longTickSubject.next(true);
     }, LONG_TICK_INTERVAL_MS);
 
+    window.setInterval(()=> {
+      this.frameSubject.next(true);
```

Comment:
I don't understand the purpose of doing an additional interval here on the same frequency as the standard tick.

Reply:
Because the standard tick calls Tick and thereby tickSubject.next on a loop.

Reply:
I think this is much faster than angular's interpolation rate, but I'm having trouble finding anything that says how often interpolation actually runs. Crazy idea: tie the frameSubject.next() to a function that gets called by the interpolator so that it actually fires on angular interpolation frames instead of on a fixed interval.